mesh/activation.go

```diff
@@ -76,6 +76,10 @@ func (t ActivationTx) Id() AtxId {
 	return AtxId{crypto.Keccak256Hash(tx)}
 }
 
+func (t ActivationTx) Validate() error {
```

Comment:
Validate should return true false - if valid or not

Reply:
I like this pattern for validation where no error means it passed and if it failed the error tells you why.  An additional boolean would just reflect that (if there's an error it should be `false`, otherwise `true`).

How is the calling code supposed to handle the result? If the `error` is not `nil` -- also check the `bool`, or just ignore it and assume it's `true`? If the `bool` is `false` and the `error` is `nil` -- I don't get a reason that I can log?